pandas_datareader/google/options.py

```diff
@@ -0,0 +1,209 @@
+                Vol: Volume traded, int64
+                Open_Int: Open interest, int64
+                IsNonstandard: True if the the deliverable is not 100 shares, otherwise false
+                Underlying: Ticker of the underlying security, string
+                Underlying_Price: Price of the underlying security, float64
+                Quote_Time: Time of the quote, Timestamp
+
+        Notes
+        -----
+        Note: Format of returned data frame is dependent on Google and may change.
+
+        When called, this function will add instance variables named
+        calls and puts. See the following example:
+
+            >>> aapl = Options('aapl', 'google')  # Create object
+            >>> aapl.calls  # will give an AttributeError
+            >>> aapl.get_options()  # Get data and set ivars
+            >>> aapl.calls  # Doesn't throw AttributeError
+
+        Also note that aapl.calls and appl.puts will always be the calls
+        and puts for the next expiry. If the user calls this method with
+        a different expiry, the ivar will be named callsYYMMDD or putsYYMMDD,
+        where YY, MM and DD are, respectively, two digit representations of
+        the year, month and day for the expiry of the options.
+
+        """
+        d = self._load_data(expiry)
+        return self._process_data(d, expiry)
+
+    @property
+    def expiry_dates(self):
+        """
+        Returns a list of available expiry dates
+        """
+        try:
+            expiry_dates = self._expiry_dates
+        except AttributeError:
+            d = self._load_data(dt.datetime.today())
+            expiry_dates = d['expirations']
+        return map(lambda d: dt.date(d['y'], d['m'], d['d']), expiry_dates)
+
+    def _load_data(self, expiry):
+        url = self._OPTIONS_BASE_URL.format(sym=self.symbol, day=expiry.day,
+                                            month=expiry.month, year=expiry.year)
+        s = re.sub(r'(\w+):', '"\\1":', self._read_url_as_StringIO(url).read())
+        return json.loads(s)
+
+    def _process_data(self, d, expiry):
+        """
+        Parse JSON data into the DataFrame
+
+        """
+        now = dt.datetime.now()
+
+        columns = ['Last', 'Bid', 'Ask', 'Chg', 'PctChg', 'Vol', 'Open_Int', 'Root', 'Underlying_Price', 'Quote_Time']
+        indexes = ['Strike', 'Expiry', 'Type', 'Symbol']
+        rows_list=[]
+        index = []
+        self._process_rows(d['puts'], rows_list, now, index, expiry, 'put', d['underlying_price'])
+        self._process_rows(d['calls'], rows_list, now, index, expiry, 'call', d['underlying_price'])
+        df=DataFrame(rows_list, columns=columns, index=MultiIndex.from_tuples(index, names=indexes))
+        return df.sort_index()
+
+    def _process_rows(self, l, rows_list, now, index, expiry, type, underlying_price):
+        for row in l:
+            dict = {}
+            try:
+                dict['Last']=float(row['p'].replace(',',''))
```

Comment:
```python
dict['Last']=float(row['p'].replace(',',''))
```

should be

```python
d['Last'] = float(row['p'].replace(',', ''))
```

(notice space because of PEP8)